simphony_metadata.yml

```diff
@@ -419,3 +419,28 @@ CUDS_KEYS:
   ELECTROSTATIC_MODEL:
     parent: CUBA.PHYSICS_EQUATION
     definition: Electrostatic model
+    physics_equation: [MOLECULAR_DYNAMICS]
+
+  BASIS:
+     definition: Space basis vectors
+     parent: CUBA.COMPONENT
+     CUBA.VECTOR:
+        shape: (3)
+        default: [[1, 0, 0], [0, 1, 0], [0, 0, 1]]
+
+  ORIGIN:
+    definition: The origin of a space system
+    parent: CUDS:COMPONENT
+    CUBA.POINT:
+      default: [0 0 0]
+
+  BOX:
+    definition: A simple hexahedron box object
+    parent: CUBA.BOUNDARY
+    CUBA.BASIS:
+      default:
+        CUBA.VECTOR: [[1, 0, 0], [0, 1, 0], [0, 0, 1]]
```

Comment:
> By primary CUBA I understand that as attributes defined in cuba.yml

I mean the value that we assign to their CUBA entry (defined in a structure) not a `default` attribute inside cuby.yaml entries.

@ahashibon @kitchoi Still the problem remains: _how can we recognize a default value for a primary CUBA from a default value for a structure_. In the `CUBA.BASIS` example, how do we understand the the given value goes into `VECTOR` attribute of the `BOX` and not another one? (a structure could have multiple attributes).